src/wallet/wallet.cpp

```diff
@@ -1110,9 +1110,9 @@ bool CWallet::AbandonTransaction(const uint256& hashTx)
         CWalletTx& wtx = it->second;
         int currentconfirm = wtx.GetDepthInMainChain();
         // If the orig tx was not in block, none of its spends can be
-        assert(currentconfirm <= 0);
```

Comment:
Its spends can very well be conflicted independently from orig tx being conflicted or not, no?

Reply:
Thanks, fixed.